examples/api/lib/cupertino/dialog/cupertino_alert_dialog.0.dart

```diff
@@ -23,9 +23,9 @@ class AlertDialogApp extends StatelessWidget {
 class AlertDialogExample extends StatelessWidget {
   const AlertDialogExample({super.key});
 
-  // This shows a CupertinoModalPopup which hosts a CupertinoAlertDialog.
+  // This shows a CupertinoAlertDialog.
```

Comment:
This comment seems unnecessary given the function name. If you named the function `_showCupertinoAlertDialog` there'd be no confusion at all :-)